src/components/atoms/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -49,20 +50,15 @@ export const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
   const isShown = show !== undefined ? show : isVisible;
 
   return (
-    <Modal
-      className="ConfirmationModal"
-      show={isShown}
-      onHide={hide}
-      centered={centered}
-    >
+    <Modal show={isShown} onHide={hide} centered={isCentered}>
       <Modal.Body>
-        <div>
-          {hasHeader && <Modal.Title>{header}</Modal.Title>}
+        <div className="ConfirmationModal">
+          {hasHeader && <h4>{header}</h4>}
           <div className="ConfirmationModal__message">{message}</div>
           <div className="ConfirmationModal__buttons">
-            <ButtonNG onClick={cancel}>{no}</ButtonNG>
-            <ButtonNG variant="primary" onClick={confirm}>
-              {yes}
+            <ButtonNG onClick={cancel}>{cancelBtnLabel}</ButtonNG>
+            <ButtonNG variant="danger" onClick={confirm}>
```

Comment:
by default, the variant should be "primary", and if "danger" is needed it can be set with the help of props.

Reply:
done a4e73af